src/main/java/io/cryostat/net/web/http/api/v1/RecordingDeleteHandler.java

```diff
@@ -98,15 +98,18 @@ public boolean isAsync() {
 
     @Override
     public void handleAuthenticated(RoutingContext ctx) throws Exception {
-        String recordingName = ctx.pathParam("recordingName");
-        try {
-            recordingArchiveHelper.deleteRecording(recordingName).get();
-            ctx.response().end();
-        } catch (ExecutionException e) {
-            if (ExceptionUtils.getRootCause(e) instanceof RecordingNotFoundException) {
-                throw new HttpException(404, e.getMessage(), e);
-            }
-            throw e;
-        }
+        // ctx.response().putHeader(HttpHeaders.LOCATION,
```

Comment:
I think you can leave this method body in-place. The `DeprecatedApi` annotation should cause the webserver to simply not call into this handler, since it'll be wrapped by a deprecation decorator handler that sets the Location header, 410 status, and ends the response early.